### PR TITLE
Delete old notices

### PR DESCRIPTION
**Pull request by esethna**
This PR removes outdated and irrelevant notices from production.

@saturninoabril @stevemudie I'm not sure who is owning notice testing as it was previously handled by Ogi. Please help add the appropriate team member for testing, thanks!

### CODE REVIEW

**Comment by saturninoabril** [comment]
I also don't know but I thought in-product notices is closer to Growth team and so maybe they are the most appropriate to own? Copying @marianunez as I could be totally wrong.

For other team member's testing, in general, I'd suggest to involve staff who implemented the feature, survey, or call to action who could also help setup the test server for QA testing and together verify if the notice works as expected.

**Comment by marianunez** [comment]
> @saturninoabril @stevemudie is there any documentation within the QA team on how to test notices or any QA members at all familiar with the process? I'm not sure how Ogi tested, and all the developers that worked on the feature are no longer at Mattermost. AFAIK there is a constantly running test server that reads the notices.json file from the `master` branch.

There is some instructions in the Readme of this repo - here: https://github.com/mattermost/notices#master

**Comment by esethna** [comment]
@marianunez ya I wrote the readme for the repo, but it doesn't include the testing steps that QA executed to verify new notices when PRs are submitted. 

AFAIK they:
- Submitted a similar PR against the master branch, merged it, then tested on a dedicated MM server for notice testing that gets the updates from the master branch notices.json within a shorter timeframe (5min or less), compared to all other servers that only check daily for new notices from https://notices.mattermost.com/ (which is based on the release branch)

**Comment by saturninoabril** [comment]
@esethna @marianunez I submitted a PR to address on how to test notices, PR submission template/requirement and QA criteria. Honestly, I'm not comfortable with the current process of making the change and passing the test solely to the QA reviewer. Quality and testing starts with the submitter like we normally do, and it should be known to be working as expected before actually creating a PR.

File: notices.json [hunk 1]
```diff
@@ -1,38 +1,4 @@
...
-  {
-    "id": "use_case_survey",
-    "conditions": {
-      "displayDate": "<= 2022-08-01T00:00:00Z"
-    },
-    "localizedMessages": {
-      "en": {
-        "title": "We want your feedback!",
-        "description": "We are looking for feedback on how we are doing in supporting your team's collaboration throughout different phases of the DevOps lifecycle. Please help us with a quick 5 minutes survey for a chance to win a set of AirPod Pros.",
-        "image": "https://raw.githubusercontent.com/mattermost/notices/master/images/Feedback.png",
```
Comment (saturninoabril) [inline notices.json:11]: Should this image be removed from the repo or this is reusable and will remain as is?

Same question for:
- https://raw.githubusercontent.com/mattermost/notices/release/images/mattermost-cloud-freemium.png
- https://raw.githubusercontent.com/mattermost/notices/master/images/v6.0.gif
- https://raw.githubusercontent.com/mattermost/notices/master/images/Boards_v6.2.gif

File: notices.json [hunk 2]
```diff
@@ -1,38 +1,4 @@
...
-        "actionText": "Give feedback",
-        "actionParam": "https://forms.gle/otcG8ETaG8URZtEt8"
```
Comment (saturninoabril) [inline notices.json:13]: I can see this form is still active. Should this be decommissioned or taken out of public reach if not to be used?

**Review by saturninoabril — APPROVED** [review]
Thanks @esethna, LGTM. Left non-blocking comment.

File: notices.json (hunk 1 above)
Comment (esethna) [reply notices.json:11]: Images are reusable so no need to remove them

**Comment by esethna** [comment]
Thanks @saturninoabril!

**Pull request merged**